kubernetes/ssl-proxy-rc.yaml

```diff
@@ -0,0 +1,56 @@
+# Replication control to run nginx-ssl-proxy
+#
+# From https://github.com/GoogleCloudPlatform/kube-jenkins-imager/blob/master/ssl_proxy.yaml
+# See license/kube_jenkins_imager_license.txt
+
+kind: ReplicationController
+apiVersion: v1
+metadata:
+  name: nginx-ssl-proxy
+  labels:
+    name: nginx-ssl-proxy
+    role: ssl-proxy
```

Comment:
For all the sources, I think it might be nice to have a label that's

```yaml
app: eups
```

That way you can select all resources from this deployment in a kubernetes namespace (say someday there's semi-related sidecar resources that end up in the namespace that you want to distinguish from resources from `sandbox-eups`).